manager/integration/tests/test_settings.py

```diff
@@ -278,47 +279,41 @@ def test_setting_toleration_extra(core_api, apps_api):  # NOQA
 def wait_for_toleration_update(core_api, apps_api, count,  # NOQA
                                expected_tolerations,
                                chk_removed_tolerations=[]):
-    not_managed_apps = [
-        "csi-attacher",
-        "csi-provisioner",
-        "csi-resizer",
-        "csi-snapshotter",
-        "longhorn-csi-plugin",
+    skip_updates = [
         "longhorn-driver-deployer",
         "longhorn-manager",
         "longhorn-ui",
     ]
-    updated = False
+    ok = False
     for _ in range(RETRY_COUNTS):
         time.sleep(RETRY_INTERVAL_LONG)
 
-        updated = True
+        ok = False
         if not check_workload_update(core_api, apps_api, count):
```

Comment:
1. It's better to rename this function to `check_longhorn_managed_workloads` to avoid misunderstanding.
2. Need to add check for new components like backing image manager pods and share manager pods.
3. Avoid checking longhorn-manager, longhorn-ui, and longhorn-driver-deployer pods.

Reply:
> 1. It's better to rename this function to `check_longhorn_managed_workloads` to avoid misunderstanding.

Addressed.

> 2. Need to add check for new components like backing image manager pods and share manager pods.

Add to create volume with backing image and this will be checked in `wait_for_toleration_update` and `wait_for_priority_class_update`.

Did not address checking for share manager because volume needs to be detached before the update and doesn't look like it will be there when doing setting updates.

> 3. Avoid checking longhorn-manager, longhorn-ui, and longhorn-driver-deployer pods.

Addressed.